package.json

```diff
@@ -20,48 +24,28 @@
     "test:types": "tsc -p types/test",
     "release": "standard-version --prerelease rc && git push --follow-tags && npm publish"
   },
-  "engines": {
-    "node": ">=6.0.0"
-  },
-  "eslintIgnore": [
-    "lib/templates/*.*"
-  ],
-  "files": [
-    "lib",
-    "types/*.d.ts"
-  ],
-  "jest": {
-    "testEnvironment": "node",
-    "collectCoverage": true
-  },
   "dependencies": {
-    "cross-fetch": "^3.0.4",
-    "universal-cookie": "^4.0.3",
-    "vue-apollo": "^3.0.4",
+    "cross-fetch": "^3.0.6",
+    "universal-cookie": "^4.0.4",
+    "vue-apollo": "^3.0.5",
     "vue-cli-plugin-apollo": "^0.21.3",
```

Comment:
The main point of the package updates is `vue-cli-plugin-apollo 0.22.~`

Reply:
> The main point of the package updates is `vue-cli-plugin-apollo 0.22.~`

My bad, i make a new commit.